app/screens/interactive_dialog/dialog_introduction_text.test.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import Preferences from 'mattermost-redux/constants/preferences';
+
+import DialogIntroductionText from './dialog_introduction_text.js';
+
+describe('DialogIntroductionText', () => {
+    const baseProps = {
+        theme: Preferences.THEMES.default,
+        value: '**bold** *italic* [link](https://mattermost.com/) <br/> [link target blank](!https://mattermost.com/)',
+    };
+
+    test('should render the introduction text correctly', () => {
+        const wrapper = shallow(
+            <DialogIntroductionText
+                {...baseProps}
+            />
+        );
+
+        expect(wrapper.getElement()).toMatchSnapshot();
+    });
+
+    test('should not fail with empty value', () => {
```

Comment:
if the value is required, does it make sense to render the component? maybe this should render `null`

Reply:
Yes, that doesn't make much sense :).  Should I push the changes?

Reply:
yes please

Reply:
PR updated